src/test/java/org/folio/edge/sip2/repositories/CirculationRepositoryTests.java

```diff
@@ -413,8 +578,8 @@ public void cannotGetLoansByUserId(Vertx vertx,
 
     final SessionData sessionData = TestUtils.getMockedSessionData();
 
-    final CirculationRepository circulationRepository =
-        new CirculationRepository(mockFolioProvider, clock);
+    final CirculationRepository circulationRepository = new CirculationRepository(
```

Comment:
Seeing repeated instantiations of new CirculationRepository(), I wonder if it's worth it to have a util method in this class to return a new CirculationRepository object. It could save us from future updates if we have to update the class's constructor.  Better yet we could hide the public constructor of CirculatioinRepository, or for any class for that matter, and expose a single method to return a new object to prevent this kind of coupling. But this is low priority...

Reply:
That'd be nice. We still need to pass the tests mocks to the object. I suppose we could use a builder for that instead. This is true of all the repository tests so I think we should use another story to come up with and implement an approach.

Reply:
Sure, we can add a todo item in the TODO jira for it.